server/src/internalClusterTest/java/org/elasticsearch/search/fetch/FetchSubPhasePluginIT.java

```diff
@@ -119,12 +119,9 @@ public void hitExecute(SearchContext context, HitContext hitContext) {
                 return;
             }
             String field = fetchSubPhaseBuilder.getField();
-            if (hitContext.hit().fieldsOrNull() == null) {
-                hitContext.hit().fields(new HashMap<>());
-            }
             DocumentField hitField = hitContext.hit().getFields().get(NAME);
             if (hitField == null) {
-                hitField = new DocumentField(NAME, new ArrayList<>(1));
+                hitField = new DocumentField(NAME, new ArrayList<>(1), context.mapperService().isMetadataField(NAME));
```

Comment:
nit: `context.mapperService().isMetadataField(NAME)`- this always seems to be `false`, from grepping the code it also seems "term_vectors_fetch" used in this test isn't used anywhere else.

Reply:
@cbuescher  Are you suggesting that instead of `context.mapperService().isMetadataField(NAME)` we can just use `false` in this test?  

Right, we can do that.